tests/wpt/css-tests/cssom-1_dev/html/index-002.htm

```diff
@@ -30,7 +30,7 @@
       ],
 
       'overflow': [
-        ['overflow-x: scroll; overflow-y: hidden;', 'overflow: scroll hidden;'],
+        ['overflow-x: scroll; overflow-y: hidden;', 'overflow-x: scroll; overflow-y: hidden;'],
```

Comment:
Note: this change will be overwritten the next time we update from upstream, since we can't push changes to it from servo's copy. See #12326.